File: src/nomanssave/bI.java
package nomanssave;

class bI implements bK {
   // $FF: synthetic field
   final bE ey;

   bI(bE var1) {
      this.ey = var1;
   }

   public String getID() {
      return "TWordsLearnt";
   }

   public boolean isSpecial() {
      return true;
   }

   public String ab() {
      return Integer.toString(bE.a(this.ey).b(eU.kt));
   }

   public void l(String var1) {
      throw new RuntimeException("Cannot set words learnt");
   }
}

File: NMSSaveEditor/nomanssave/mixed/bI.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace NMSSaveEditor
{


public class bI : bK {
   public bE ey;

   public bI(bE var1) {
      this.ey = var1;
   }

   public string getID() {
      return "TWordsLearnt";
   }

   public bool isSpecial() {
      return true;
   }

   public string ab() {
      // PORT_TODO: return (bE.a(this.ey).ToString().b(eU.kt));
      return default;
   }

   public void l(string var1) {
      throw new Exception("Cannot set words learnt");
   }
}



}
